vip-helpers/vip-elasticsearch.php

```diff
@@ -450,4 +450,16 @@ function wpcom_search_api_wp_to_es_args( $args ) {
 	}
 
     return $es_query_args;
-}
\ No newline at end of file
+}
+
+/**
+ * Filter ElasticPress index name if using VIP ES infrastructure
+ */
+function filter_ep_index_name( $index_name, $blog_id, $indexables ) {
```

Comment:
Can we use a more unique function name? (or even an anonymous function).

Reply:
Changed in 7187875a860f2a742d2bbcac6cc4760300aecdfc